tasks/install-FreeBSD.yml

```diff
@@ -4,6 +4,14 @@
   pkgng:
     name: rubygem-fluentd
     state: present
+  register: register_fluentd_rubygem_fluentd
+
+- name: Patch rc.d script
+  patch:
```

Comment:
but by the time the system is deployed, we should be using our own package tree. it's a hack from the start and i would like to keep the patch for inclusion.